README.md

```diff
@@ -200,7 +200,7 @@ The keys for the A+ course settings are listed below:
 ```
 
 Some fields require a value from specific choices (see also
-[the MOOC-grader documentation about the index.yaml file](https://github.com/Aalto-LeTech/mooc-grader/blob/master/courses/README.md)):
+[the MOOC-grader documentation about the index.yaml file](https://github.com/apluslms/mooc-grader/blob/master/courses/README.md)):
```

Comment:
These fixes should be in a separate commit. They are not related to the substitution feature.

Reply:
It's actually causing a git conflict since the Github links have already been fixed in the upstream.

Reply:
Good to know this didn't help merging pull requests. I will revert to README.md to d2dd81f and re-apply my changes there.